iml-services/iml-action-runner/src/local_actions.rs

```diff
@@ -119,11 +121,23 @@ pub async fn handle_local_action(
                 spawn_plugin(fut, in_flight, id);
 
                 rx.await.map_err(ActionRunnerError::OneShotCanceledError)
-            } else {
-                Err(ActionRunnerError::RequiredError(error::RequiredError(
-                    format!("Could not find action {} in local registry", action),
-                )))
             }
-        }
+            "snapshot_mount" | "snapshot_unmount" => {
+                let rx = add_in_flight(Arc::clone(&in_flight), id.clone()).await;
+                let args2 = args.clone();
+
+                let fut = wrap_plugin(args, move |fqdn: Fqdn| {
+                    let (_, fut) = invoke_rust_agent(fqdn, action, args2);
```

Comment:
This will ignore cancellation, which needs to be handled.

Reply:
Done